### PR TITLE
DOCSUP-10532: Edited and translated to Russian (dateName and bitPositionsToArray functions)

### PR DESCRIPTION
**Pull request by gyuton**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category (leave one):
- Documentation for #25372 and #25394

### CODE REVIEW

File: docs/en/sql-reference/functions/date-time-functions.md [hunk 1]
```diff
@@ -1014,7 +1015,7 @@ Result:
 
 ## dateName {#dataname}
 
-Returns part of date with specified date part.
+Returns specified part of the date.
```
Comment (olgarev) [inline docs/en/sql-reference/functions/date-time-functions.md:1018]:
```suggestion
Returns the specified part of date.
```

File: docs/en/sql-reference/functions/date-time-functions.md [hunk 2]
```diff
@@ -1024,13 +1025,13 @@ dateName(date_part, date)
...
--   `date` — Date [Date](../../sql-reference/data-types/date.md) or DateTime [DateTime](../../sql-reference/data-types/datetime.md), [DateTime64](../../sql-reference/data-types/datetime64.md).
-
+-   `date_part` — Date part. Possible values: `year`, `quarter`, `month`, `week`, `dayofyear`, `day`, `weekday`, `hour`, `minute`, `second`. [String](../../sql-reference/data-types/string.md).
+-   `date` — Date. [Date](../../sql-reference/data-types/date.md), [DateTime](../../sql-reference/data-types/datetime.md) or [DateTime64](../../sql-reference/data-types/datetime64.md).
+-   `timezone` — Timezone. Optional. [String](../../sql-reference/data-types/string.md).
 
 **Returned value**
 
--   Specified date part of date.
+-   Specified part of the date.
```
Comment (olgarev) [inline docs/en/sql-reference/functions/date-time-functions.md:1034]:
```suggestion
-   The specified part of date.
```

File: docs/en/sql-reference/functions/date-time-functions.md [hunk 3]
```diff
@@ -1024,13 +1025,13 @@ dateName(date_part, date)
 
 **Arguments**
 
--   `date_part` - Date part. Possible values .
--   `date` — Date [Date](../../sql-reference/data-types/date.md) or DateTime [DateTime](../../sql-reference/data-types/datetime.md), [DateTime64](../../sql-reference/data-types/datetime64.md).
-
+-   `date_part` — Date part. Possible values: `year`, `quarter`, `month`, `week`, `dayofyear`, `day`, `weekday`, `hour`, `minute`, `second`. [String](../../sql-reference/data-types/string.md).
```
Comment (olgarev) [inline docs/en/sql-reference/functions/date-time-functions.md:1028]:
```suggestion
-   `date_part` — Date part. Possible values: 'year', 'quarter', 'month', 'week', 'dayofyear', 'day', 'weekday', 'hour', 'minute', 'second'. [String](../../sql-reference/data-types/string.md).
```

File: docs/en/sql-reference/functions/encoding-functions.md [hunk 4]
```diff
@@ -224,7 +224,7 @@ Accepts an integer. Returns an array of UInt64 numbers containing the list of po
 
 ## bitPositionsToArray(num) {#bitpositionstoarraynum}
 
-Accepts an integer, argument will be converted to unsigned integer type. Returns an array of UInt64 numbers containing the list of positions of bits that equals 1. Numbers in the array are in ascending order.
+Accepts an integer and converts it to an unsigned integer. Returns an array of `UInt64` numbers containing the list of positions of bits of `arg` that equals `1` in ascending order.
```
Comment (olgarev) [inline docs/en/sql-reference/functions/encoding-functions.md:227]:
```suggestion
Accepts an integer and converts it to an unsigned integer. Returns an array of `UInt64` numbers containing the list of positions of bits of `arg` that equal `1`, in ascending order.
```

File: docs/en/sql-reference/functions/encoding-functions.md [hunk 5]
```diff
@@ -234,11 +234,13 @@ bitPositionsToArray(arg)
 
 **Arguments**
 
--   `arg` — Integer value.Types:  [Int/UInt](../../sql-reference/data-types/int-uint.md)
+-   `arg` — Integer value. [Int/UInt](../../sql-reference/data-types/int-uint.md).
 
 **Returned value**
 
-An array of UInt64 numbers containing the list of positions of bits that equals 1. Numbers in the array are in ascending order.
+-   An array containing a list of positions of bits that equals `1` in ascending order.
```
Comment (olgarev) [inline docs/en/sql-reference/functions/encoding-functions.md:241]:
```suggestion
-   An array containing a list of positions of bits that equal `1`, in ascending order.
```

File: docs/ru/sql-reference/functions/date-time-functions.md [hunk 6]
```diff
@@ -27,40 +27,40 @@ SELECT
...
 
-**Синтаксис** 
+**Синтаксис**
 
 ``` sql
 timeZone()
 ```
 
-Псевдоним: `timezone`. 
+Псевдоним: `timezone`.
```
Comment (olgarev) [inline docs/ru/sql-reference/functions/date-time-functions.md:36]:
```suggestion
Синоним: `timezone`.
```

Comment (olgarev) [inline docs/ru/sql-reference/functions/date-time-functions.md:36]:
```suggestion
Синоним: `timezone`.
```

File: docs/ru/sql-reference/functions/date-time-functions.md [hunk 7]
```diff
@@ -102,21 +103,21 @@ int32samoa: 1546300800
...
 
-**Синтаксис** 
+**Синтаксис**
 
 ``` sql
 timeZoneOf(value)
 ```
 
-Псевдоним: `timezoneOf`. 
+Псевдоним: `timezoneOf`.
```
Comment (olgarev) [inline docs/ru/sql-reference/functions/date-time-functions.md:112]:
```suggestion
Синоним: `timezoneOf`.
```

File: docs/ru/sql-reference/functions/date-time-functions.md [hunk 8]
```diff
@@ -145,15 +146,15 @@ SELECT timezoneOf(now());
 timeZoneOffset(value)
 ```
 
-Псевдоним: `timezoneOffset`. 
+Псевдоним: `timezoneOffset`.
```
Comment (olgarev) [inline docs/ru/sql-reference/functions/date-time-functions.md:149]:
```suggestion
Синоним: `timezoneOffset`.
```

File: docs/ru/sql-reference/functions/date-time-functions.md [hunk 9]
```diff
@@ -1025,6 +1026,45 @@ SELECT formatDateTime(toDate('2010-01-04'), '%g');
...
+
+**Синтаксис**
+
+``` sql
+dateName(date_part, date)
+```
+
+**Аргументы**
+
+-   `date_part` — часть даты. Возможные значения: `year`, `quarter`, `month`, `week`, `dayofyear`, `day`, `weekday`, `hour`, `minute`, `second`. [String](../../sql-reference/data-types/string.md).
```
Comment (olgarev) [inline docs/ru/sql-reference/functions/date-time-functions.md:1041]:
```suggestion
-   `date_part` — часть даты. Возможные значения: 'year', 'quarter', 'month', 'week', 'dayofyear', 'day', 'weekday', 'hour', 'minute', 'second'. [String](../../sql-reference/data-types/string.md).
```

File: docs/ru/sql-reference/functions/date-time-functions.md [hunk 10]
```diff
@@ -1025,6 +1026,45 @@ SELECT formatDateTime(toDate('2010-01-04'), '%g');
...
+-   `date` — дата. [Date](../../sql-reference/data-types/date.md), [DateTime](../../sql-reference/data-types/datetime.md) или [DateTime64](../../sql-reference/data-types/datetime64.md).
+-   `timezone` — часовой пояс. Необязательный параметр. [String](../../sql-reference/data-types/string.md).
```
Comment (olgarev) [inline docs/ru/sql-reference/functions/date-time-functions.md:1043]:
```suggestion
-   `timezone` — часовой пояс. Необязательный аргумент. [String](../../sql-reference/data-types/string.md).
```

File: docs/ru/sql-reference/functions/date-time-functions.md [hunk 11]
```diff
@@ -1025,6 +1026,45 @@ SELECT formatDateTime(toDate('2010-01-04'), '%g');
...
+
+**Возвращаемое значение**
+
+-   Указанная часть даты.
+
+Тип: [String](../../sql-reference/data-types/string.md#string)
```
Comment (olgarev) [inline docs/ru/sql-reference/functions/date-time-functions.md:1049]:
```suggestion
Тип: [String](../../sql-reference/data-types/string.md#string).
```

File: docs/ru/sql-reference/functions/encoding-functions.md [hunk 12]
```diff
@@ -223,3 +223,53 @@ SELECT reinterpretAsUInt64(reverse(unhex('FFF'))) AS num;
...
+
+**Синтаксис**
+
+```sql
+bitPositionsToArray(arg)
+```
+
+**Аргументы**
+
+-   `arg` — целое значения. [Int/UInt](../../sql-reference/data-types/int-uint.md).
```
Comment (olgarev) [inline docs/ru/sql-reference/functions/encoding-functions.md:239]:
```suggestion
-   `arg` — целое значение. [Int/UInt](../../sql-reference/data-types/int-uint.md).
```

File: docs/ru/sql-reference/functions/encoding-functions.md [hunk 13]
```diff
@@ -223,3 +223,53 @@ SELECT reinterpretAsUInt64(reverse(unhex('FFF'))) AS num;
...
+
+**Возвращаемое значение**
+
+-   Массив, содержащий список позиций битов, равных `1`, в порядке возрастания.
+
+Тип: [Array](../../sql-reference/data-types/array.md)([UInt64](../../sql-reference/data-types/int-uint.md)).
+
+**Example**
```
Comment (olgarev) [inline docs/ru/sql-reference/functions/encoding-functions.md:247]:
```suggestion
**Пример**
```

File: docs/ru/sql-reference/functions/date-time-functions.md [hunk 14]
```diff
@@ -27,40 +27,40 @@ SELECT
...
+Переводит дату или дату с временем в указанный часовой пояс. Часовой пояс - это атрибут типов `Date` и `DateTime`. Внутреннее значение (количество секунд) поля таблицы или результирующего столбца не изменяется, изменяется тип поля и, соответственно, его текстовое отображение.
 
-**Синтаксис** 
+**Синтаксис**
 
 ``` sql
 toTimezone(value, timezone)
 ```
 
 Псевдоним: `toTimezone`.
```
Comment (olgarev) [inline docs/ru/sql-reference/functions/date-time-functions.md:54]:
```suggestion
Синоним: `toTimezone`.
```

**Comment by gyuton** [comment]
@kitaisreal Hi! Can you review this, please?

**Review by kitaisreal — APPROVED** [review]

**Pull request merged**